setup.py

```diff
@@ -46,7 +46,7 @@ def readme():
 
 
 extras = dict(
-    geography=["GeoAlchemy2", "shapely"],
+    geography=["GeoAlchemy2==0.12.5", "shapely"],
```

Comment:
It is not clear why we included a specific version of `GeoAlchemy2` in this PR.
Ari is not supporting this task, so I am unable to ask about it.
I am gonna remove it and ensure that tests pass without it.

If tests pass, I recommend we merge.